### ISSUE TITLE
Errors in command line handlebars should be caught.

### ISSUE THREAD

**Issue by PeterCamilleri** [PeterCamilleri/mysh#15]
Right now an error in a command line handlebar aborts:
```
mysh>$w = {{ (Dir.pwd) }}
Set value w = C:/Sites/mysh
mysh>$w = {{ decorate(Dir.pwd) }}
C:/Sites/mysh/lib/mysh/expression.rb:23:in `initialize': undefined method `decorate' for #<#<Class:0x1c85418>:0x1c85388> (NoMethodError)
        from C:/Sites/mysh/lib/mysh/expression.rb:52:in `eval'
        from C:/Sites/mysh/lib/mysh/expression.rb:52:in `mysh_eval'
        from C:/Sites/mysh/lib/mysh/user_input/handlebars.rb:38:in `block in do_process_handlebars'
        from C:/Sites/mysh/lib/mysh/user_input/handlebars.rb:35:in `gsub'
        from C:/Sites/mysh/lib/mysh/user_input/handlebars.rb:35:in `do_process_handlebars'
        from C:/Sites/mysh/lib/mysh/user_input/handlebars.rb:26:in `eval_handlebars'
        from lib/mysh.rb:24:in `run'
        from lib/mysh.rb:57:in `<main>'
```

**Comment by PeterCamilleri** [comment]
Fixed in the add_variables branch.

**Comment by PeterCamilleri** [comment]
Fixed in version 0.3.0
